src/components/Blocks/Introduction/Edit.jsx

```diff
@@ -2,13 +2,30 @@ import React from 'react';
 import { SidebarPortal } from '@plone/volto/components';
 import Sidebar from './Sidebar';
 import { DetachedTextBlockEditor } from '@plone/volto-slate/blocks/Text/DetachedTextBlockEditor';
+import config from '@plone/volto/registry';
 
 const IntroductionBlockEdit = (props) => {
   const { data, onChangeBlock, block, selected } = props;
+  const unwantedButtons = ['heading-two', 'heading-three', 'blockquote'];
+
+  const filteredToolbarProps = {
+    ...props,
+    slateSettings: {
+      ...config.settings.slate,
+      toolbarButtons: config.settings.slate.toolbarButtons
+        .filter((button) => !unwantedButtons.includes(button))
+        // eslint-disable-next-line array-callback-return
+        .filter((button, index, array) => {
+          if (button !== array[index + 1]) {
+            return button;
+          }
+        }),
```

Comment:
@danalvrz You can simplify this and avoid the eslint rule like this:

```suggestion
        // remove any adjacent separators
        .filter((button, index, array) => button !== array[index + 1]),
```

I also added a comment to remind us later what this is for.